CI/travis/before_install_darwin

```diff
@@ -10,4 +10,5 @@ brew_install_or_upgrade() {
 }
 
 brew_install_or_upgrade libmatio
-brew_install_or_upgrade cppcheck
\ No newline at end of file
+brew_install_or_upgrade cppcheck
+brew_install_or_upgrade astyle
```

Comment:
at this point, it's a good idea to switch to a for loop
have a list of packages and iterate
```
PACKAGES="libmatio cppcheck astyle"
for package in $PACKAGES ; do
    brew_install_or_upgrade "$package"
done
```